eventdata/challenges/bulk-size-evaluation.json

```diff
@@ -111,5 +111,59 @@
       "clients": {{ p_bulk_indexing_clients }}
     }
   ]
+},
+{
+  "name": "bulk-size-concurrency-evaluation",
```

Comment:
As it is implemented now, this is prone to ordering bias. The first combination will hit an empty index but the application is not yet properly warmed-up, the second one is somewhere in the middle and by the time the last one will run, the application is probably warmed-up but the index has already grown so IMHO it is hard to compare the results. Also, I am not so sure about setting the warmup time period to zero.

Reply:
Would adding a minute warmup and recreating the index ahead of each step suffice?

Reply:
With enough warmup and recreation of indices I suspect ordering bias would no longer be a factor.